aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,346 @@
+# `crossplane` PostgresDB example
+
+In this more complex example, we're going to walk through the process of
+enabling provider-families with `sts:AssumeRoleWithWebIdentity` to the
+management cluster, then using that to bootstrap a database using a secondary
+role assumed by crossplane for the actual build.
+
+> **Warning** Creating IAM permissions
+>
+> The  following document assumes that you have permissions to create IAM roles
+> and policies inside your account.
+>
+> If you do not have such permissions, please reach out to your account
+> administrator and ask them to create these for you.
+
+## Create the primary role
+
```

Comment:
Brief intro? like "We are creating a role `crossplane-assume-role` for crossplane controller to let it assume the specific provider families roles where the permission for RDS are... "